### PR TITLE
Add script to populate song collection in database

### PR DESCRIPTION
**Pull request by daxkerchner**
(For running on dev desktops)

I also moved the MongoDB URI to a separate file.  This file is now referenced by app.js and my script.

## Testing

cleared database song collection and ran `node bin/dev-setup` - output:
```
Saved sample song "Amazing Grace" with id 582e227b5d45682508aa450e
Saved translated song "Sublime gracia del Señor" with id 582e227b5d45682508aa450f
Closing DB connection
```

And verified the songs in the DB and the UI.

To test the error path I replaced the line `sampleSong.save().then(` with
```javascript
sampleSong.save().then(
  function(song) {
    throw song.author;
  }
).then
```

When I ran the script, the output was
```
Unable to store sample song. Reason:
John Newton and Virginia Harmony
Unable to store translated song. Reason:
Unable to store sample song
Closing DB connection
```

### CODE REVIEW

File: bin/dev-setup [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  timeAdded: new Date()
+});
+
+mongoose.connect(mongoConfig.uri);
+
+sampleSong.save().then(
+  function(song) {
+    console.log('Saved sample song "' + song.title + '" with id ' + song.id);
+    translatedSong.source = song.id;
+    return translatedSong.save(translatedSong)
```
Comment (jonathanarde) [inline bin/dev-setup]: do you need to provide the argument?

File: bin/dev-setup [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    'That saved a wretch like me!',
+    'I once was lost, but now am found;',
+    'Was blind, but now I see.'],
+    ['The earth shall soon dissolve like snow;',
+    'The sun forbear to shine;',
+    'But God, who called me here below,',
+    'Will be forever mine.']
+  ],
+  source: null,
+  oriSong: null,
```
Comment (jonathanarde) [inline bin/dev-setup]: i think we dont need this field anymore. i already put a github issue regarding this

File: bin/dev-setup [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    'Que a mi pecador salvó',
+    'Fui ciego mas hoy miro yo',
+    'Perdido y el me amó.'],
+    ['[ No tenemos',
+    'una traducción',
+    'para este',
+    'verso :( ]']
+  ],
+  source: null, // will be changed
+  oriSong: 'Amazing Grace',
```
Comment (jonathanarde) [inline bin/dev-setup]: same as my previous comment

File: bin/dev-setup (hunk 1 above)
Comment (daxkerchner) [inline bin/dev-setup]: Oops, you're right

**Comment by daxkerchner** [comment]
Updated based on pr comments

**Pull request merged**